### PR TITLE
Private Path for Files for Multisite

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes: #5845 

## Summary

**[Private Paths for Files and Code](https://pantheon.io/docs/private-paths)** - Updated to show the private path for files in a WordPress multisite. [PR 5935](https://github.com/pantheon-systems/documentation/pull/5935)

## Post Launch

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/private-paths.md [hunk 1]
```diff
@@ -3,31 +3,44 @@ title: Private Paths for Files and Code
...
 
-Alternatively, you can store sensitive data in a JSON or ini-style text file within the `wp-content/uploads/private` (WordPress) or `sites/default/files/private` (Drupal) directories. These directories are symbolically linked to Valhalla and can also be accessed from the `files` directory when connecting via SFTP. This allows secure data to be distributed to other environments, while keeping it out of version control. You can then read the data from `settings.php` or `wp-config.php`, like so:
+Alternatively, you can store sensitive data in a JSON or ini-style text file within the following directories:
 
-```php
+- `wp-content/uploads/private` (WordPress)
+- `/wp-content/uploads/private/sites/blog_id/` (WordPress Multisite)
+- `sites/default/files/private` (Drupal)
+
+These directories are symbolically linked to Valhalla and can also be accessed from the `files` directory when connecting via SFTP. This allows secure data to be distributed to other environments, while keeping it out of version control. You can then read the data from `settings.php` or `wp-config.php`, like so:
```
Comment (alexfornuto) [inline source/content/private-paths.md:41]: For the technical review, this is the relevant change. The rest is copy updates.

Comment (jocastaneda) [reply source/content/private-paths.md:41]: Looks good and read well.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5935](https://pr-5935--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `69` | `69` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `93` | `93` | `0` |

Lighthouse Audit Reports:

- [`pr-5935` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23098/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23098/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5935--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by tmountjr** [comment]
Only concern is that `/wp-content/uploads/private/sites/blog_id/` kinda looks like the actual path. Can we do `/wp-content/uploads/private/sites/<blog_id>/` instead to indicate that `blog_id` is variable?

**Pull request merged**